packages/appstore/src/constants/utils.ts

```diff
@@ -106,3 +71,40 @@ export const getWalletHeaderButtons = (is_demo: boolean, handleAction?: () => vo
               },
           ];
 };
+
+type TAccount = {
```

Comment:
@george-usynin-deriv This type could be moved to the types folder in the appstore